docs/fidesops/docs/guides/connection_types.md

```diff
@@ -31,3 +31,32 @@ database options and third party API services with which Fidesops can communicat
     "size": 50
 }
 ```
+
+
+## Required Connection Secrets
+
+To view the secrets needed to authenticate with a given connection, visit `/api/v1/connection_type/<connection_type>/secret`.
```

Comment:
Can we specify HTTP method? Looks like we don't throughout this file, but I think it makes sense, even if we specify it in the example code.

Reply:
good call